src/components/PostList/PostList.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './PostList.scss';
+
+import { Post } from '../Post';
+import { User } from '../User';
+import { CommentList } from '../CommentList';
+import { userPostCommentType } from '../../types';
+
+export const PostList = ({ userPostsComments }) => (
+  <ul>
+    {userPostsComments.map(post => (
+      <li className="full-post" key={post.id}>
+        <Post post={post} />
+        <User post={post} />
+        <CommentList post={post} key={post.comment.postId} />
```

Comment:
pass comments, not the whole post object
key is not needed here, you've already added it to the li

```suggestion
        <CommentList comments={comments}/>
```